subsys/bluetooth/mesh/shell.c

```diff
@@ -2000,13 +2540,22 @@ int cmd_timeout(const struct shell *shell, size_t argc, char *argv[])
 		}
 
 		bt_mesh_cfg_cli_timeout_set(timeout_ms);
+		bt_mesh_health_cli_timeout_set(timeout_ms);
```

Comment:
Please change the description for this command in doc/reference/bluetooth/mesh/shell.rst to show that this now also changes the health client timeout